control/bdalg.py

```diff
@@ -221,18 +222,18 @@ def feedback(sys1, sys2=1, sign=-1):
     """
 
     # Check for correct input types.
-    if not isinstance(sys1, (int, float, complex, tf.TransferFunction,
-                             ss.StateSpace, frd.FRD)):
+    if not isinstance(sys1, (int, float, complex, np.number,
```

Comment:
I'm struggling to find docs on numpy.number; it has a __doc__ string, but no entry, e.g., here: https://docs.scipy.org/doc/numpy-1.13.0/genindex.html .  This use seems reasonable, just not sure how standard it is.

Reply:
Numpy scalar type hierarchy is here: https://docs.scipy.org/doc/numpy-1.13.0/reference/arrays.scalars.html